packages/proteus/src/proteus/errors/DecryptError.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Wire
+ * Copyright (C) 2016 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/* eslint no-magic-numbers: "off" */
+
+const ProteusError = require('./ProteusError');
+
+/** @module errors */
+
+/**
+ * @extends ProteusError
+ * @param {string} [message]
+ * @param {string} [code]
+ */
+class DecryptError extends ProteusError {
+  constructor(message = 'Unknown decryption error', code = 2) {
+    super(message, code);
+  }
+
+  static get CODE() {
+    return {
+      CASE_200: 200,
+      CASE_201: 201,
+      CASE_202: 202,
+      CASE_203: 203,
+      CASE_204: 204,
+      CASE_205: 205,
+      CASE_206: 206,
+      CASE_207: 207,
+      CASE_208: 208,
+      CASE_209: 209,
+      CASE_210: 210,
+      CASE_211: 211,
+      CASE_212: 212,
```

Comment:
Very clear cases.. :) @bennyn any chance improving this area?

Reply:
It would require changes in webapp as well and I don't want to do it at this point in time.